### ISSUE TITLE
v1.56 `--daemon` no longer forks to background on armv7/amd64

### ISSUE THREAD

**Issue by sedlund** [rclone/rclone#5472]
#### What is the problem you are having with rclone?

```
rclone mount --daemon -vvv b2crypt: b2crypt        
2021/07/21 21:53:18 DEBUG : rclone: Version "v1.56.0" starting with parameters ["rclone" "mount" "--daemon" "-vvv" "b2crypt:" "b2crypt"]
2021/07/21 21:53:18 DEBUG : Creating backend with remote "b2crypt:"
2021/07/21 21:53:18 DEBUG : Using config file from "/home/pi/.config/rclone/rclone.conf"
2021/07/21 21:53:18 DEBUG : Creating backend with remote "b2:sedlund-rcrypt"
```
**DOES NOT FORK**

v1.55.1

```
rclone mount --daemon -vvv b2crypt: b2crypt   
2021/07/21 21:58:16 DEBUG : Using config file from "/home/pi/.config/rclone/rclone.conf"
2021/07/21 21:58:16 DEBUG : rclone: Version "v1.55.1" starting with parameters ["rclone" "mount" "--daemon" "-vvv" "b2crypt:" "b2crypt"]
2021/07/21 21:58:16 DEBUG : Creating backend with remote "b2crypt:"
2021/07/21 21:58:16 DEBUG : Creating backend with remote "b2:sedlund-rcrypt"
2021/07/21 21:58:17 DEBUG : rclone: Version "v1.55.1" finishing with parameters ["rclone" "mount" "--daemon" "-vvv" "b2crypt:" "b2crypt"]
$
```
Forks and I get my prompt back.

#### What is your rclone version (output from `rclone version`)

v1.56 

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)

32bit Raspberry Pi OS Buster

#### Which cloud storage system are you using? (e.g. Google Drive)

Backblaze B2

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)

see log above

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)

See Abve

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by sedlund** [comment]
Tested on amd64/Ubuntu 21.04 also does not fork

```
rclone mount --daemon -vvv b2crypt: b2crypt
2021/07/21 17:20:06 DEBUG : rclone: Version "v1.56.0" starting with parameters ["rclone" "mount" "--daemon" "-vvv" "b2crypt:" "b2crypt"]
2021/07/21 17:20:06 DEBUG : Creating backend with remote "b2crypt:"
2021/07/21 17:20:06 DEBUG : Using config file from "/home/sedlund/.config/rclone/rclone.conf"
2021/07/21 17:20:06 DEBUG : Creating backend with remote "b2:sedlund-rcrypt"
```

**Comment by ivandeex** [comment]
bisect found bad commit: https://github.com/rclone/rclone/commit/221dfc38825769a4911ed5b21e1760d3adf9bc88

@sedlund thank you for report. will fix this bug in 1.57.1

**Comment by ivandeex** [comment]
hi @sedlund 
please check whether this patched rclone build fixes your issue
https://github.com/ivandeex/rclone/releases/tag/v1.56.0-bgmount01

**Comment by sedlund** [comment]
Yes it does.  Thanks!

```
─ rclone mount --daemon -vvv b2crypt: test    
2021/07/24 15:13:33 DEBUG : Using config file from "/home/sedlund/.config/rclone/rclone.conf"
2021/07/24 15:13:33 DEBUG : rclone: Version "v1.55.1" starting with parameters ["rclone" "mount" "--daemon" "-vvv" "b2crypt:" "test"]
2021/07/24 15:13:33 DEBUG : Creating backend with remote "b2crypt:"
2021/07/24 15:13:33 DEBUG : Creating backend with remote "b2:sedlund-rcrypt"
2021/07/24 15:13:34 DEBUG : rclone: Version "v1.55.1" finishing with parameters ["rclone" "mount" "--daemon" "-vvv" "b2crypt:" "test"]
╭─    ~/Downloads/rclone-v1.56.0-bgmount01-linux-amd64    dev ?6    
```

**Comment by infused-kim** [comment]
Works for me too. Thank you.

**Comment by ivandeex** [comment]
Fix merged and will be released with 1.561.
Test build removed.
